### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by nuriman1**

### CODE REVIEW

**Comment by nuriman1** [comment]
ready

**Comment by thaodtd-0665** [comment]
commented
just testing :p

**Comment by thaodtd-0665** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (manhnt-0960) [inline Gemfile]: remove all comment

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
+# Use SCSS for stylesheets
+gem 'sass-rails', '>= 6'
+# Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
+gem 'webpacker', '~> 4.0'
```
Comment (manhnt-0960) [inline Gemfile]: use `"` instead of `'`

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: use `"` instead of `'`

Comment (nuriman1) [reply app/views/layouts/application.html.erb]: I have resolved this for line 5 to 8. But may I know, other than line 5 to 8, do I need to use " instead ' for other lines too ?

**Comment by nuriman1** [comment]
ready

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.1'
+
+
+gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
+# Use Puma as the app server
+gem 'puma', '~> 4.1'
```
Comment (saobv-2223) [inline Gemfile]: remove all comment lines and using " instead of '

File: app/javascript/packs/application.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: using ' please

File: app/mailers/application_mailer.rb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
```
Comment (saobv-2223) [inline app/mailers/application_mailer.rb]: using " in ruby file

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: using "

File: config/credentials.yml.enc [hunk 8]
```diff
@@ -0,0 +1 @@
+Xh84L3iV3L+XdDLLIhddxjrMDGzvnp7FIGegxr2WzGOU37OkY5uK5NmQiQhGI5hkFJQLg5x7RUsKH6xFaxSs8jJBzTbpkEB7ZbCUgLBokyUmLiRvTljOAZWScUzLDd9ZL+mVNnojL0zevoxY+VlJhDnbWltGfMEXJ/fcsE4Wjlsdh3mq0KT6ihfz18bsVthXLoCr0DtD1MWxsSQVRpiGbc4YjeP8FHZqxKK/nNHZ1ro/b0OOZcwRjyGQRI4Z238TNPp64YED6TeCKiGwoOplIM677ADdFgo7tTO5Tf9SnKfpzZ7fKSvE+2K/v0uK7CAYEwmk4GetDMuorufWBFvSf8HSLWfhpeCoe9DJFY5F/M711jTXgEeQta18IR0n7QHtyPBV56VR/FAt8+1IVGr1pHcnHj4/EyORGbrk--TJ5bKNd6jR/UIf9m--U3lT0rEk2CtT8rYljAk7aQ==
```
Comment (saobv-2223) [inline config/credentials.yml.enc:1]: create `credentials.yml.enc.example` and push to git, this file add to gitignore

File: config/database.yml [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
```
Comment (saobv-2223) [inline config/database.yml]: do same as `credentials.yml.enc` file

**Review by saobv-2223 — COMMENTED** [review]
you PR is having 2 commits, merge to 1 commit, please

**Comment by nuriman1** [comment]
ready

File: .gitignore [hunk 10]
```diff
@@ -1,56 +1,43 @@
...
+/config/master.key
+
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
+/config/database.yml 
+/config/credentials.yml.enc
```
Comment (saobv-2223) [inline .gitignore]: image

File: Gemfile [hunk 11]
```diff
@@ -0,0 +1,48 @@
+source "https://rubygems.org"
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby "2.7.1"
+
+
```
Comment (saobv-2223) [inline Gemfile]:
```suggestion
```

File: Gemfile [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+gem "sass-rails", ">= 6"
+
+gem "webpacker", "~> 4.0"
+
+gem "turbolinks", "~> 5"
+
+gem "jbuilder", "~> 2.7"
+
+
+
```
Comment (saobv-2223) [inline Gemfile]:
```suggestion
```

File: Gemfile [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+gem "bootsnap", ">= 1.4.2", require: false
+
+group :development, :test do
+  
+  gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+end
+
+group :development do
+
```
Comment (saobv-2223) [inline Gemfile]:
```suggestion
```

File: config/credentials.yml.enc (hunk 8 above)
Comment (saobv-2223) [inline config/credentials.yml.enc:1]: image

File: test/application_system_test_case.rb [hunk 14]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
```
Comment (saobv-2223) [inline test/application_system_test_case.rb]: image

**Comment by nuriman1** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**